rules/android/lint/providers.bzl

```diff
@@ -11,6 +11,8 @@ AndroidLintNodeInfo = provider(
         updated_baseline = "The updated baseline XML",
         lint_result_xml = "The lint results XML file",
         lint_junit_xml = "Lint result formatted in Junit format",
+        lint_checks = "Custom Lint Targets",
```

Comment:
Is it worth making putting this into separate provider? `AndroidLintInspectInfo` perhaps?

Reply:
can we put a provider in another provder?

Reply:
Yeah can, accepted pattern. We already do it for passing transitive lint node infos.